src/api/mutations/payment/usePostPaymentOptionMutation.ts

```diff
@@ -0,0 +1,34 @@
+import toast from 'react-hot-toast';
+import { useNavigate } from 'react-router-dom';
+import { useMutation } from '@tanstack/react-query';
+import httpClient from '@/api/httpClient';
```

Comment:
Just `@` symbol isn't a good choise as alias for root directory of the project, it is better to give meaningful name like `@capstone` or `@src` at least